packages/component-library/src/components/plmg-text-input/test/plmg-text-input.spec.tsx

```diff
@@ -0,0 +1,145 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { TextInput } from '../plmg-text-input';
+
+describe('plmg-text-input', () => {
+  it('renders default', async () => {
+    const page = await newSpecPage({
+      components: [TextInput],
+      html: `<plmg-text-input label="Default"></plmg-text-input>`,
+    });
+    expect(page.root).toEqualHtml(`
+
+    <plmg-text-input label="Default">
+    <div class="plmg-text-input-wrapper">
+    <label class="medium plmg-text-input-label" htmlfor="default">
+    Default
+    </label>
+      <div class="plmg-text-input-field-wrapper" tabindex="0">
+        <input class="medium" id="default" type="text" name="Default">
+      </div>
+    </plmg-text-input>
```

Comment:
Try to appropriately tab your expected html for easier readability.  
```suggestion
    <plmg-text-input label="Hidden" show-label="false">
      <div class="plmg-text-input-wrapper" />
      <label
        class="medium plmg-text-input-label visually-hidden"
        htmlfor="hidden"
      >
        Hidden
      </label>
      <div class="plmg-text-input-field-wrapper" tabindex="0">
        <input class="medium" id="hidden" type="text" name="Hidden" />
      </div>
    </plmg-text-input>
```

Tip: You can copy the html into the component html and let prettier do the hard work for you.